finetune/target_models/sequence_labeling.py

```diff
@@ -44,10 +40,13 @@ def text_to_tokens_mask(self, X, Y=None, context=None):
             if Y is None:
                 yield feats
             if Y is not None:
+                filtered_labels = [
```

Comment:
I think this might re-compute the min / max for every label?  Nbd either way though

Reply:
Yeah it does.  Not a big deal either way but seems reasonable to move the min / max out of the list comp